cfgmgr/vxlanmgr.cpp

```diff
@@ -432,6 +452,27 @@ bool VxlanMgr::isVxlanStateOk(const std::string & vxlanName)
     return false;
 }
 
+std::string VxlanMgr::getSwitchMacAddress()
```

Comment:
Can you please rename this to` getVxlanRouterMacAddress()`. There is a different `SwitchMacAddress `and it would be confusing with the name.

Reply:
Thank you for you suggestion, Done it.